src/main/java/com/github/Anon8281/universalScheduler/foliaScheduler/FoliaScheduledTask.java

```diff
@@ -4,11 +4,20 @@
 import io.papermc.paper.threadedregions.scheduler.ScheduledTask;
 import org.bukkit.plugin.Plugin;
 
+import java.util.HashMap;
+import java.util.Random;
+
 public class FoliaScheduledTask implements MyScheduledTask {
     private final ScheduledTask task;
+    static final HashMap<Integer, ScheduledTask> tasks = new HashMap<>();
```

Comment:
HashMap is not thread safe.

I pretty sure someone will catch an unexpected error here due the folia threads

Reply:
I'm pretty sure that there is no requirement to say that if HashMap is not thread-safe, then ConcurrentHashMap is.

if we talking about thread safe and the map, I would like to use
```
private static volatile ConcurrentHashMap<Integer, ScheduledTask> tasks = new ConcurrentHashMap<Integer, ScheduledTask>();
```

However, it looks like the [PLATFORM]ScheduledTask is exactly the task itself, and we actually shouldn't store here more than 1 task info.

Maybe it would be better to move this part to the FoliaScheduler?